cluster-manifests/region1/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+- ../base/
+commonAnnotations:
+  region: "1"
+patchesStrategicMerge:
+- regionalize-flux-git-path.yaml
```

Comment:
Did this file not get checked in by chance?

Reply:
we are modifying them from [2a12](https://github.com/mspnp/aks-secure-baseline/pull/183/commits/2a12cc40214b78ae9cbaa24412fa8a51cc9fcdc4), so they are now in for sure.

Reply:
Good ol' json patches :)  Thanks.